fontscan/rune_coverage.go

```diff
@@ -289,6 +289,38 @@ func (s *scriptSet) insert(newScript language.Script) {
 	(*s)[scriptIdx] = newScript
 }
 
+const scriptSize = 4
+
+// serialize serialize the script set in binary format
+func (ss scriptSet) serialize() []byte {
+	buffer := make([]byte, 2+scriptSize*len(ss))
```

Comment:
I don't follow why this uses `2+scriptSize*len(ss)`. It only uses one byte to store the length, so what's the other byte for?

Reply:
Oh, I see you fixed this already. Sorry!